demos/python_demos/image_inpainting_demo/image_inpainting_demo.py

```diff
@@ -33,30 +34,103 @@ def build_argparser():
                       help="Optional. Specify the target device to infer on; CPU, GPU, FPGA, HDDL or MYRIAD is "
                            "acceptable. The demo will look for a suitable plugin for device specified. "
                            "Default value is CPU", default="CPU", type=str)
-    args.add_argument("-p", "--parts", help="Optional. Number of parts to draw mask.", default=8, type=int)
-    args.add_argument("-mbw", "--max_brush_width", help="Optional. Max width of brush to draw mask.", default=24, type=int)
-    args.add_argument("-ml", "--max_length", help="Optional. Max strokes length to draw mask.", default=100, type=int)
-    args.add_argument("-mv", "--max_vertex", help="Optional. Max number of vertex to draw mask.", default=20, type=int)
-    args.add_argument("--no_show", help="Optional. Don't show output", action='store_true')
+    args.add_argument("-r", "--rnd",
+                      help="Optional. Use random mask for inpainting (with parameters set by -p, -mbw, -mk and -mv).Skipped in GUI mode",
+                      action='store_true')
+    args.add_argument("-p", "--parts", help="Optional. Number of parts to draw mask. Skipped in GUI mode", default=8, type=int)
+    args.add_argument("-mbw", "--max_brush_width", help="Optional. Max width of brush to draw mask. Skipped in GUI mode",
+                      default=24, type=int)
+    args.add_argument("-ml", "--max_length", help="Optional. Max strokes length to draw mask. Skipped in GUI mode",
+                      default=100, type=int)
+    args.add_argument("-mv", "--max_vertex", help="Optional. Max number of vertex to draw mask. Skipped in GUI mode",
+                      default=20, type=int)
+    args.add_argument("-mc", "--mask_color",
+                      help="Optional. Color to be treated as mask (provide 3 RGB components in range of 0...255)."
+                      " Default is 0 0 0. Skipped in GUI mode", default=[0, 0, 0], type=int, nargs="+")
+    args.add_argument("--no_show", help="Optional. Don't show output. Skipped in GUI mode", action='store_true')
+    args.add_argument("-o", "--output", help="Optional. Save output to the file with provided filename. Skipped in GUI mode",
+                      default="", type=str)
+    args.add_argument("-a", "--auto", help="Optional. Use automatic (non-interactive) mode instead of GUI", action='store_true')
 
     return parser
 
-def main():
-    args = build_argparser().parse_args()
+
+def loadImage(imgPath):
```

Comment:
```python
def loadImage(imgPath):
...
def build_argparser():
```
You should use only one style for whole demo. In this case, camel style is prefferable, but [PEP 8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names) insists on snake style.

Reply:
> You should use only one style for whole demo.

I think in the case of Python demos we can make a stronger statement: you should use the standard Python style for the demo.